README.md

```diff
@@ -78,17 +85,45 @@ file { '/etc/someservice/db.yaml':
 
 This module provides the `conjur::secret` function, described above, and the `conjur` class, which can be configured to establish Conjur host identity on the node running Puppet.
 
-### Pre-established host identity
+### Conjur host identity with Host Factory
```

Comment:
I think we should have `Conjur host identity with API key` first. I don't think HFT is more common in general than hardcoded creds per host.